capstone/src/main/webapp/script.js

```diff
@@ -26,8 +20,142 @@ function createMap() {
     });
   
   cityLimit.setMap(map);
+
+  //**button for centering map*/
+  let centerControlDiv = document.createElement("div");
+  let centerControl = new CenterControl(centerControlDiv, map);
+  centerControlDiv.index = 1;
+  map.controls[google.maps.ControlPosition.LEFT_CENTER].push(centerControlDiv);
+
+  //**button for zipcode data layer */
```

Comment:
If you are up-to-date with master, could you deploy this live so we can see how the buttons work? :)

Reply:
I'm not, but it's just a simple on/off switch